File: CS/DocumentManager_TabbedUI/Form1.cs
using System;
using System.Windows.Forms;
using DevExpress.XtraBars.Docking2010;
using DevExpress.XtraBars.Docking2010.Views.Tabbed;
using DevExpress.XtraEditors;

namespace DocumentManager_TabbedUI {
    public partial class Form1 : Form {
        public Form1() {
            InitializeComponent();
        }
        void Form1_Load(object sender, EventArgs e) {
            AddDocumentManager();
            for(int i = 0; i < 3; i++) {
                AddChildForm();
            }
        }
        void AddDocumentManager() {
            DocumentManager manager = new DocumentManager();
            manager.MdiParent = this;
            manager.View = new TabbedView();
        }
        int count;
        void AddChildForm() {
            Form childForm = new Form();
            childForm.Text = "Child Form " + (++count).ToString();

            SimpleButton btn = new SimpleButton();
            btn.Text = "Button " + count.ToString();
            btn.Parent = childForm;

            childForm.MdiParent = this;
            childForm.Show();
        }
    }
}

File: VB/DocumentManager_TabbedUI/Form1.vb
Imports Microsoft.VisualBasic
Imports System
Imports System.Windows.Forms
Imports DevExpress.XtraBars.Docking2010
Imports DevExpress.XtraBars.Docking2010.Views.Tabbed
Imports DevExpress.XtraEditors

Namespace DocumentManager_TabbedUI
	Partial Public Class Form1
		Inherits Form
		Public Sub New()
			InitializeComponent()
		End Sub
		Private Sub Form1_Load(ByVal sender As Object, ByVal e As EventArgs) Handles MyBase.Load
			AddDocumentManager()
			For i As Integer = 0 To 2
				AddChildForm()
			Next i
		End Sub
		Private Sub AddDocumentManager()
			Dim manager As New DocumentManager()
			manager.MdiParent = Me
			manager.View = New TabbedView()
		End Sub
		Private count As Integer
		Private Sub AddChildForm()
			Dim childForm As New Form()
			count += 1
			childForm.Text = "Child Form " & (count).ToString()

			Dim btn As New SimpleButton()
			btn.Text = "Button " & count.ToString()
			btn.Parent = childForm

			childForm.MdiParent = Me
			childForm.Show()
		End Sub
	End Class
End Namespace